epictrack-web/src/components/workPlan/status/StatusContainer.tsx

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { Box, Grid } from "@mui/material";
+import { ETHeading3 } from "../../shared";
+import { Palette } from "../../../styles/theme";
+import { makeStyles } from "@mui/styles";
+import StatusView from "./StatusView";
+import { WorkplanContext } from "../WorkPlanContext";
+import { ETTab, ETTabs } from "../../shared/tab/Tab";
+import RichTextEditor from "../../shared/richTextEditor";
+import TabPanel from "../../shared/tab/TabPanel";
+
+const useStyle = makeStyles({
+  title: {
+    borderBottom: `2px solid ${Palette.primary.main}`,
+    paddingBottom: "0.5rem",
+  },
+  tab: {
+    paddingBottom: "0.5rem !important",
+    fontSize: "1.5rem !important",
+    fontWeight: 400,
+    lineHeight: "1.3rem",
+    "&.Mui-selected": {
+      fontWeight: 400,
+    },
+    minHeight: "0px",
+  },
+  tabPanel: {
+    padding: "1.5rem 0px 1rem 1rem",
+    minHeight: "0px",
+  },
+});
+
+const StatusContainer = () => {
+  const ctx = React.useContext(WorkplanContext);
+  const classes = useStyle();
+  const [selectedTabIndex, setSelectedTabIndex] = React.useState(0);
+  const status = React.useMemo(() => ctx.status, [ctx.status]);
```

Comment:
I don't think we need a useMemo, we can just pull the status from the context as needed. It's better if we destruct the context as well. Instead of ctx = we do { status } =